tests/migrations.rs

```diff
@@ -114,3 +115,69 @@ fn check_non_null_column_without_default() {
         }
     }
 }
+
+fn get_up_migrations() -> Vec<PathBuf> {
+    let root = env!("CARGO_MANIFEST_DIR");
+    let migrations_dir = PathBuf::from(root).join("migrations");
+
+    let mut migrations = Vec::new();
+    for entry in std::fs::read_dir(migrations_dir).unwrap() {
+        let path = entry.unwrap().path();
+
+        if path.is_file() && path.to_string_lossy().contains(".up.sql") {
```

Comment:
```suggestion
        if path.is_file() && path.ends_with(".up.sql") {
```

Reply:
Sorry, this was a bad advice, I didn't realize that `ends_with` only checks the last component, so this doesn't work. But at least it helped me find a loophole in the tests - right now all tests are green even if you remove all test data files, but the tests don't actually find any migrations :sweat_smile: Let's add this:

```rust
#[test]
fn check_migrations_have_sample_data() {
    let migrations = get_up_migrations();
    assert!(!migrations.is_empty());
    for migration_path in migrations {
```